<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Molecular biology</title>
  <style>
    html {
      line-height: 1.5;
      font-family: unset;
      font-size: 20px;
      color: #1a1a1a;
      background-color: #fdfdfd;
    }
    body {
      margin: 0 auto;
      max-width: 36em;
      padding-left: 50px;
      padding-right: 50px;
      padding-top: 50px;
      padding-bottom: 50px;
      hyphens: auto;
      overflow-wrap: break-word;
      text-rendering: optimizeLegibility;
      font-kerning: normal;
    }
    @media (max-width: 600px) {
      body {
        font-size: 0.9em;
        padding: 1em;
      }
      h1 {
        font-size: 1.8em;
      }
    }
    @media print {
      body {
        background-color: transparent;
        color: black;
        font-size: 12pt;
      }
      p, h2, h3 {
        orphans: 3;
        widows: 3;
      }
      h2, h3, h4 {
        page-break-after: avoid;
      }
    }
    p {
      margin: 1em 0;
    }
    a {
      color: #1a1a1a;
    }
    a:visited {
      color: #1a1a1a;
    }
    img {
      max-width: 100%;
    }
    h1, h2, h3, h4, h5, h6 {
      margin-top: 1.4em;
    }
    h5, h6 {
      font-size: 1em;
      font-style: italic;
    }
    h6 {
      font-weight: normal;
    }
    ol, ul {
      padding-left: 1.7em;
      margin-top: 1em;
    }
    li > ol, li > ul {
      margin-top: 0;
    }
    blockquote {
      margin: 1em 0 1em 1.7em;
      padding-left: 1em;
      border-left: 2px solid #e6e6e6;
      color: #606060;
    }
    code {
      font-family: Menlo, Monaco, 'Lucida Console', Consolas, monospace;
      font-size: 85%;
      margin: 0;
    }
    pre {
      margin: 1em 0;
      overflow: auto;
    }
    pre code {
      padding: 0;
      overflow: visible;
      overflow-wrap: normal;
    }
    .sourceCode {
     background-color: transparent;
     overflow: visible;
    }
    hr {
      background-color: #1a1a1a;
      border: none;
      height: 1px;
      margin: 1em 0;
    }
    table {
      margin: 1em 0;
      border-collapse: collapse;
      width: 100%;
      overflow-x: auto;
      display: block;
      font-variant-numeric: lining-nums tabular-nums;
    }
    table caption {
      margin-bottom: 0.75em;
    }
    tbody {
      margin-top: 0.5em;
      border-top: 1px solid #1a1a1a;
      border-bottom: 1px solid #1a1a1a;
    }
    th {
      border-top: 1px solid #1a1a1a;
      padding: 0.25em 0.5em 0.25em 0.5em;
    }
    td {
      padding: 0.125em 0.5em 0.25em 0.5em;
    }
    header {
      margin-bottom: 4em;
      text-align: center;
    }
    #TOC li {
      list-style: none;
    }
    #TOC ul {
      padding-left: 1.3em;
    }
    #TOC > ul {
      padding-left: 0;
    }
    #TOC a:not(:hover) {
      text-decoration: none;
    }
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
  </style>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.21/dist/katex.min.css" crossorigin="anonymous">
  <style>
    header { margin-bottom: 2em; }
    body { max-width: 30em; }

    a:not(h1 a), a:visited:not(h1 a) { color: blue; }

    h1 > a { text-decoration: none; }

    .katex { font-size: 1em; }

    .katex-display {
      overflow-x: auto;
      padding-block: 10px; /* vertical scroll bug */
    }

    @media (max-width: 600px) {
      .katex-display {
        /* overcome padding on mobile */
        margin-left: -1em;
        margin-right: -1em;
      }
    }  </style>
</head>
<body>
<header id="title-block-header">
<h1 class="title"><a href="/">Molecular biology</a></h1>
<!--<p class="date">Aug 2024</p>-->
</header>
<h3 id="mrna">mRNA</h3>
<p>Scale, in ascending order: atom &lt; molecule &lt; nucleotide/nucleic
acid/base which all refer somewhat to ACGT in DNA (ACGU in RNA,
ribonucleic acid) &lt; codon (3 RNA bases). Hence there are 64 possible
codons, that encode 20 amino acids (molecular compounds).</p>
<p>That encoding is called the genetic code. It is <em>not</em> a simple
scheme of one combination of 3 bases (possibly with repetition) mapping
to a single amino acid, <a
href="https://gist.github.com/guillaumekoenig/a9ba49446721504bc1d021a7a6f6c90c#file-combi_rep-py-L10">even
though that produces 20 possibilities too</a>. It was cracked
experimentally by synthesizing chains of codons and looking at the
resulting amino acid chains. AUG is the start sequence codon, UAG the
stop sequence codon.</p>
<p>Ribosomes translate RNA into chains of amino acids. The resulting
chains fold on themselves in 3D: those are the proteins. They perform
biological functions.</p>
<p>The human genome has 3 billion base pairs in 23 chromosomes (6
billion when accounting for homologous pairs of chromosomes). <em>E.
coli</em>'s has 4.5 million base pairs in one chromosome.</p>
<p>Phenotype: physical expression of traits, shaped by genes and the
environment. Genotype: expression of traits in the genes.</p>
<p>mRNA is the kind that DNA is copied into and brought into the
ribosome (m for messenger). Then codons need to be translated into amino
acids: that's the role of tRNA, molecules having the <em>anticodon</em>
that matches the codon (by base-pairing) and holding the associated
amino acid. mRNA is short-lived (minutes). Ribosomal RNA (discussed
later) is much more prevalent, and long-lived.</p>
<p>It was experimentally verified that codons come in 3 bases the
following way. Insert a single base, and the resulting protein is
deadly. Same if you insert 2 bases. However insert 3, and you get a
mostly functional protein.</p>
<h3 id="introns-and-splicing">Introns and splicing</h3>
<p>DNA is translated into mRNA in the nucleus (by base pairing), before
it is exported to the cytoplasm for building the protein. In the
nucleus, it's referred to as precursor mRNA to distinguish it from when
it's in the cytoplasm. Early on it was clear that E. coli's mRNA was the
same throughout. But not so in larger animals: precursor mRNA was on
average 10 times larger than the final one in the cytoplasm. How
come?</p>
<p>On the adenovirus (common cold), the mRNA in the cytoplasm shows
parts that are spliced out compared to its source DNA. The non-coding
DNA parts are called introns.</p>
<ul>
<li>Yeast: 6000 protein-coding genes</li>
<li>Humans: 24000 protein-coding genes (even though much more complex
than yeast!), hundreds of types of cells</li>
</ul>
<p>Introns allow to build protein variants. For example, introns can
serve to alter the end of a protein: a variant with a greasy end (to
attach) vs another variant with a sugary end (to float freely). Humans
then have ~100k different kinds of proteins.</p>
<p>Nuclei have 6 varieties of small strands of RNA dubbed U1..U6 (rich
in uracil). It was found those pair up with the beginning of known
introns, allowing to cut at that spot. They are snRNA (small nuclear
RNA), and are non-coding (neither are tRNA nor ribosomal RNA).</p>
<p>The hemoglobin protein carries oxygen in the blood. It's made of 4
amino acid chains: 2 alpha-globin chains, and 2 beta-globin chains.
Sickle cell disease is due to a mutation on the beta-globin chain. But
beta-thalassemia isn't caused by a mutation. Instead it's due to a
mutation in the intron, that makes the spot look like an end marker.</p>
<h3 id="catalyst">Catalyst</h3>
<p>A catalyst accelerates a reaction. It can be a chemical catalyst (for
example baking soda) accelerating a chemical reaction, or a biological
catalyst (called an enzyme) accelerating a biochemical reaction. An
early example of enzyme known in the 19th century is zymase, which turns
sugar into alcohol + CO<sub>2</sub> (in other words it enables
fermentation).</p>
<p>Crystallizing a substance makes it pure (reducing it to crystals).
Doing so with a particular protein in a catalyzing substance, and
observing the result still has its catalyzing property, leads to the
belief that all enzymes are proteins. Note that proteins are always the
result of biological processes. (Proteins haven't been found outside
Earth, only amino acids.) They break down when boiled.</p>
<p>But not all enyzmes are proteins. That's what Cech got the Nobel for:
finding they can also be pure RNA. Unclear: how is the RNA enyzme
"activated", once isolated, in their experiments?</p>
</body>
</html>
